web/src/app/util/FilterContainer.js

```diff
@@ -37,6 +37,21 @@ export default class FilterContainer extends React.PureComponent {
     anchorEl: null,
   }
 
+  // todo: fill in rest
```

Comment:
todo?

Reply:
yeah, whoever wrote the original didn't add propTypes

Reply:
So should we remove the comment or is there work to be done?

Reply:
There is work to be done, not in scope of this PR.

Reply:
It's probably not worth the noise then (there's a bunch of places missing propTypes). I suggest we just ignore it or fix it by adding the few missing props.